80s Game/Assets/Scripts/Multiplayer/PlayerInputWrapper.cs

```diff
@@ -81,36 +88,43 @@ private void OnFire()
         }   
     }
 
+    //Received through the Unity Input system
     private void OnRecenter(InputValue value)
     {
         player.RecenterCursor();
     }
 
+    // Override for joycons
     private void OnRecenter()
     {
         player.RecenterCursor();
     }
 
+    // Received through the Unity input system
```

Comment:
I feel like method groups like this might be better off having one comment for the whole block, rather than the same comment pasted onto each one separately

Reply:
I'd rather not group by where we get the input from but rather group the methods by functionality. Makes editing in bulk easier down the line.